server/datastore/mysql/software.go

```diff
@@ -312,94 +347,140 @@ var dialect = goqu.Dialect("mysql")
 // listSoftwareDB returns all the software installed in the given hostID and list options.
 // If hostID is nil, then the method will look into the installed software of all hosts.
```

Comment:
Update docs for `listSoftwareDB` (there's no `hostID *uint` argument anymore)

Reply:
fixed in 2b4446091